### PR TITLE
WIP: AEROGEAR-8989 - Showcase app config

### PR DESCRIPTION
**Pull request by jstaffor**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-8989

### CODE REVIEW

**Comment by jstaffor** [comment]
@austincunningham  - ready for a review :+1:

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 1]
```diff
@@ -136,6 +136,79 @@ NOTE: This step also creates a variant as described in xref:push-notifications.a
 
 . xref:data-sync.adoc#sync-server-binding[Bind the {mobile-client} to the {sync-service} app] if you want to demonstrate {sync-service}.
 
-include::../generic-obtaining-the-mobile-sdk-config-file.adoc[leveloffset=1]
+//[[downloading-the-mobile-services-configuration-file]]
+= Downloading the Mobile Services Configuration File
+
+. Open your {mobile-client} in Mobile Developer Console.
+. Copy the `mobile-services.json` configuration to your clipboard.
+. Save the contents of the clipboard to a new file called `mobile-services.json`.
```
Comment (austincunningham) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: This is for the showcase app,  which will have a mobile-service.js file already in git https://github.com/aerogear/ionic-showcase/blob/master/src/mobile-services.js , don't think there is any need to create mobile-service.json.  Replace the  contents of the config object in mobile-services.js with the contents of mobile-service.json. i.e. everything between the {} inclusive . This is really hard to describe.

Comment (jstaffor) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: I don't follow @austincunningham - would you mind making this technical change and I'll review it from a docs point of view (if there is a need)?

Comment (jstaffor) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: Actually, I think I know what you are asking. Let me give it a go.

**Comment by jstaffor** [comment]
@austincunningham - ready for reivew

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc (hunk 1 above)
Comment (austincunningham) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: As the show case is only an ionic app, there is no need for the configs for the other platforms,

Comment (jstaffor) [reply modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: Given that there is no need for the configs for the other platforms, I've removed them. In addition to this, I've added a note to this section to make it clear to the reader that this section is only for the Ionic platform.

That addresses your feedback. Can you review it again please?

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 2]
```diff
@@ -136,6 +136,28 @@ NOTE: This step also creates a variant as described in xref:push-notifications.a
...
 
-include::../generic-obtaining-the-mobile-sdk-config-file.adoc[leveloffset=1]
+//[[downloading-the-mobile-services-configuration-file]]
+= Downloading the Mobile Services Configuration File
+
+NOTE: This section is only for the Ionic platform.
+
+. Open your {mobile-client} in Mobile Developer Console.
+. Copy the `mobile-services.json` configuration to your clipboard.
+. Save the contents of the clipboard to a new file called `mobile-services.json`.
```
Comment (austincunningham) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: remove this line 
```suggestion
 Save the contents of the clipboard to a new file called `mobile-services.json`.
```

**Comment by jstaffor** [comment]
@finp - this is ready for a final review

**Comment by jstaffor** [comment]
Just a note: I think this work is related to https://github.com/aerogear/mobile-docs/issues/551

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 3]
```diff
@@ -136,6 +136,28 @@ NOTE: This step also creates a variant as described in xref:push-notifications.a
 
 . xref:data-sync.adoc#sync-server-binding[Bind the {mobile-client} to the {sync-service} app] if you want to demonstrate {sync-service}.
 
-include::../generic-obtaining-the-mobile-sdk-config-file.adoc[leveloffset=1]
+//[[downloading-the-mobile-services-configuration-file]]
+= Downloading the Mobile Services Configuration File
+
+NOTE: This section only applies to the Ionic platform.
```
Comment (finp) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]: let's rewrite this, it's true... but kinda contradicted below

**Comment by finp** [comment]
@austincunningham  does this make sense for ionic?
image

or is it just that it doesn't make sense for showcase?

**Comment by jstaffor** [comment]
After discussing on docs stand-up, this work is now parked.

**Comment by finp** [comment]
Update: the documentation issue for ionic will be resolved by https://github.com/aerogear/aerogear-js-sdk/issues/318
I'm not sure when this will land, but ask "does it make sense to avoid updating ionic docs until then?"

**Pull request merged**